src/xo-app/index.js

```diff
@@ -37,6 +38,18 @@ import Vm from './vm'
 export default class XoApp extends Component {
   componentDidMount () {
     this.refs.bodyWrapper.style.minHeight = this.refs.menu.getWrappedInstance().height + 'px'
+    const previousDisclaimer = cookies.get('previousDisclaimer')
```

Comment:
I don't know how to handle the different versions because I am not the one who takes care of it, it is something we decide together.

Either way, isolating this piece of code in a function would make sense.